commons/all/src/main/java/org/infinispan/commons/util/ImmutableListCopy.java

```diff
@@ -22,12 +22,9 @@
 /**
  * A lightweight, read-only copy of a List.  Typically used in place of the common idiom: <code> return
  * Collections.unmodifiableList(new ArrayList( myInternalList )); </code>
- * <p/>
- * a it is far more efficient than making a defensive copy and then wrapping the defensive copy in a read-only wrapper.
- * <p/>
- * Also used whenever a read-only reference List is needed.
- * <p/>
- *
+  * a it is far more efficient than making a defensive copy and then wrapping the defensive copy in a read-only wrapper.
```

Comment:
Looks like we want to reword this line as well. Maybe just 
```suggestion
  * It is more efficient than making a defensive copy and then wrapping the defensive copy in a read-only wrapper.
```

Reply:
Don